ibis/backends/flink/registry.py

```diff
@@ -63,18 +72,20 @@ def _filter(translator: ExprTranslator, op: ops.Node) -> str:
     return f"CASE WHEN {bool_expr} THEN {true_expr} ELSE {false_null_expr} END"
 
 
-def _literal(translator: ExprTranslator, op: ops.Literal) -> str:
-    return translate_literal(op)
-
+def _timestamp_diff(translator: ExprTranslator, op: ops.Node) -> str:
+    left = translator.translate(op.left)
+    right = translator.translate(op.right)
+    return f"timestampdiff(second, {left}, {right})"
 
-def _zeroifnull(translator: ExprTranslator, op: ops.Literal) -> str:
-    casted = translate_literal(ops.Literal("0", dtype=op.dtype))
-    return f"COALESCE({translator.translate(op.arg)}, {casted})"
 
+def _timestamp_from_unix(translator: ExprTranslator, op: ops.Node) -> str:
+    arg, unit = op.args
 
-def _nullifzero(translator: ExprTranslator, op: ops.Literal) -> str:
-    casted = translate_literal(ops.Literal("0", dtype=op.dtype))
-    return f"NULLIF({translator.translate(op.arg)}, {casted})"
+    if unit == TimestampUnit.MILLISECOND:
+        return f"TO_TIMESTAMP_LTZ({helpers.quote_identifier(arg.name, force=True)}, 3)"
+    elif unit == TimestampUnit.SECOND:
+        return f"TO_TIMESTAMP_LTZ({helpers.quote_identifier(arg.name, force=True)}, 0)"
```

Comment:
Nit - these lines look identical except for the 3/0 as the 2nd arg. Could you refactor to reduce the code duplication?

Reply:
Done! I think it probably copies something similar from, say, `mssql` backend.

Reply:
(But it didn't feel clear enough value to reduce the duplication on that backend, from what I saw.)

Reply:
Thanks! And yeah, no need to fixup existing code now; just trying to do small cleanups as new code is added.